components/AnimateChildrenIn.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { View } from 'react-native';
+import { Transitioning, Transition } from 'react-native-reanimated';
+
+export default ({ children = {} }) => {
+  const [loaded, setLoaded] = useState(false);
+  const ref = useRef();
+
+  useEffect(() => {
+    ref.current.animateNextTransition();
+    setLoaded(true);
+  }, []);
```

Comment:
Since there are no dependencies for this, we might want to remove the extra `[]` argument.

Reply:
so passing an empty array means that this will run _only once_ on component render

passing in an array of dependencies means that this will run when these dependencies change

if we don’t pass anything, it’ll run on _every_ render

Reply:
https://reactjs.org/docs/hooks-effect.html